clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -308,11 +318,11 @@ object GarbageCollector {
     val expiredAddresses = gc
       .getExpiredAddresses(repo, runID, gcCommitsLocation, numRangePartitions, numAddressPartitions)
       .toDF("address")
-      .withColumn("run_id", lit(runID))
+      .withColumn(MARK_ID_KEY, lit(markId))
```

Comment:
This moves a known output file, so we should document it.

Reply:
of course. It's all should be documented